src/components/Overlay/Overlay.tsx

```diff
@@ -89,16 +104,19 @@ Overlay.propTypes = {
 
 export default Overlay
 
+type MUpStyle = {
+  maxWidth?: number | string
+  marginTop?: number | string
+  marginBottom?: number | string
+}
```

Comment:
Maybe we should type this more generically as React.CSSProperties, since it's spread into the styles object? 🤔

Reply:
yeah this was to match the `PropTypes` down in the files